src/api/graphItemTypes.ts

```diff
@@ -123,7 +123,7 @@ export interface VizNodeInfo {
     x: number;
     y: number;
     layoutable: boolean;
-    selected: boolean;
+    selected?: boolean; // in expand, we have to pass VizNode to ogma without selected attribute
```

Comment:
You touched the rest-client you claimed ownership @maxime-allex 

(I will make selected not mandatory also in server :p)